index.html

```diff
@@ -14,29 +14,18 @@
 	<link href="css/achilles.css" rel="stylesheet">
 	<link href="css/chart.css" rel="stylesheet">
 
-	<script>
-		var report = 'dashboard';
-		var page_vm = {};
-
-		function updateReport(value) {
-			report = value;
-			updateRoute();
-		}
-
-		function setDatasource(index) {
-			page_vm.datasource(page_vm.datasources[index]);
-			updateRoute();
-		}
-
-		function updateRoute() {
-			$('.reportDrilldown').addClass('hidden');
-			document.location = '#/' + page_vm.datasource().folder + '/' + report;
-		}
-	</script>
-
 	<script src="js/curl.config.js"></script>
 	<script src="js/curl.js"></script>
+	<script src="js/jquery-1.10.2.js"></script>
 	<script src="js/app/achilles.js"></script>
+	<script src="js/app/achillesWrapper.js"></script>	
+	<script>
+		var datasourcepath = 'data/datasources.json'
+		var containerName = "#achillesContainer";
+		var achillespath = "achilles.html";
+		var a = new Achilles( datasourcepath, containerName, achillespath, achillesWeb);
```

Comment:
I believe this should be loaded through the AMD loader.  When the code in the <script> tag gets fired is not predictable when using AMD.  Instead, this coudl be in achilles.js

Also, we've moved recently from curl.js to require.js.  curl.js should be fine for this app, but we might consider standardizing on require.js.